### PR TITLE
modified some tips of the round

### PR DESCRIPTION
**Pull request by NIXC**
We don't have most antags that are on the tip of the round. This should cause the actually interesting tips to show up more often.
Added a few ftl related tips as well.
:cl: NINXE
tweak: Modified the tip of the round to have less antags
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -47,6 +44,7 @@ As a Roboticist, you can repair your cyborgs with a welding tool. If they have t
 As the AI, you can click on people's names to look at them. This only works if there are cameras that can see them.
 As the AI, you can quickly open and close doors by holding shift while clicking them, bolt them when holding ctrl, and even shock them while holding alt.
 As the AI, you can take pictures with your camera and upload them to newscasters.
+As the AI, your default lawset is crewsmov, which applys to anyone on the crew roster.
```
Comment (ninjanomnom) [inline strings/tips.txt]: applies

File: strings/tips.txt [hunk 2]
```diff
@@ -57,9 +55,8 @@ As the Chief Engineer, you can spy on and even forge PDA communications with the
 As an Engineer, the supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you.
 As an Engineer, you can electrify grilles by placing wire "nodes" beneath them: the big seemingly unconnected bulges from a half completed wiring job.
 As an Engineer, return to Engineering once in a while to check on the singularity or energy ball and the SMES cells. It's always a good idea to make sure containment isn't compromised.
-As an Engineer, you can power the station solely with the solar arrays. While uninteresting, it is a much safer alternative to the singularity engine.
+As an Engineer, you can power the whole ship using P.A.C.M.A.N. generators. Great for when the engine explodes!
 As an Engineer, you can repair windows by using a welding tool on them while on any intent other than harm.
-As an Engineer, your construction permit allows you to create a new area, but has only one use.
```
Comment (ninjanomnom) [inline strings/tips.txt]: Why is this removed?

File: strings/tips.txt [hunk 3]
```diff
@@ -112,80 +109,8 @@ As a Shaft Miner, the northern side of Lavaland has a lot more rare minerals tha
...
-As the Malfunctioning AI, look into flooding the station with plasma fires to kill off large portions of the crew, letting you pick off the remaining few with space suits who escaped.
-As an Alien, your melee prowess is unmatched, but your ranged abilities are sorely lacking. Make use of corners to force a melee confrontation!
-As an Alien, you take double damage from all burn attacks, such as lasers, welding tools, and fires. Furthermore, fire can destroy your resin and eggs. Expose areas to space to starve away any flamethrower fires before they can do damage!
-As an Alien, resin floors not only regenerate your plasma supply, but also passively heal you. Fight on resin floors to gain a home turf advantage!
-As an Alien, the facehugger is by far your most powerful weapon because of its ability to instantly win a fight. Remember however that certain helmets, such as biohoods or space helmets will completely block facehugger attacks.
-As an Alien, you are unable to pick up or use any human items or machinery. Instead, you should focus on sabotaging APCs, computers, cameras and either stowing, spacing, or melting any weapons you find.
```
Comment (ninjanomnom) [inline strings/tips.txt]: non humanoid antag tips should be left in

File: strings/tips.txt (hunk 2 above)
Comment (NIXC) [inline strings/tips.txt]: Is that even a real thing?

File: strings/tips.txt (hunk 3 above)
Comment (NIXC) [inline strings/tips.txt]: I heard ayyyos may still be in the game

File: strings/tips.txt (hunk 2 above)
Comment (ninjanomnom) [inline strings/tips.txt]: yes

Comment (NIXC) [inline strings/tips.txt]: Readded it but I accidentally placed it a few lines down so this change thing wont pick it up. Woops
